cmd/skywire-visor/commands/root.go

```diff
@@ -262,6 +268,33 @@ func runVisor(conf *visorconfig.V1) {
 			conf.Hypervisors = append(conf.Hypervisors, pubkey)
 		}
 	}
+	//autopeering should only happen when there is no local or remote hypervisor set in the config.
+	if conf.Hypervisors != nil {
+		if conf.Hypervisor != nil {
+			autopeer = false
+		}
+	}
+	if autopeer {
+		log.Infof("autopeering...")
+		var hvkey string
+		hvkey, err := script.Exec(peercmd).String() //TODO: mrpalide this needs to run again on loss of connection to the hypervisor within the same constraints
+		if err != nil {
+			log.Error("error autopeering")
```

Comment:
Please use a full sentence here or follow the same pattern you use elsewhere.

Reply:
accidentally included now removed